src/main/java/com/example/ludogoriesoft/lukeriaerpapi/repository/ProductRepository.java

```diff
@@ -3,12 +3,15 @@
 import com.example.ludogoriesoft.lukeriaerpapi.models.Package;
 import com.example.ludogoriesoft.lukeriaerpapi.models.Product;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 import java.util.List;
 import java.util.Optional;
 
 public interface ProductRepository extends JpaRepository<Product, Long> {
     List<Product> findByDeletedFalse();
+    @Query("SELECT p FROM Product p WHERE p.forSale=true AND p.availableQuantity > 0 ")
```

Comment:
Don't really thing you need to use a query for this method, you can use Spring Data JPA's method query derivation feature, it should look something like this: List<Product> findByForSaleTrueAndAvailableQuantityGreaterThan(int quantity);